fl4health/reporting/json_reporter.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+import json
+import uuid
+from logging import INFO
+from pathlib import Path
+from typing import Any
+
+from flwr.common.logger import log
+
+from fl4health.reporting.base_reporter import BaseReporter
+
+
+class DateTimeEncoder(json.JSONEncoder):
```

Comment:
It was an artifact from the old metrics reporter. I changed all the reporting to convert the datetimes to strings before reporting them instead of only when dumping. I'm not sure if there was a reason to have the actual datetime objects in the reports dictionary but didn't want to have to add a check in the wandb reporter looking for invalid types and converting them.

Reply:
I'm going to delete this class for now, but if there are any reasons the datetime values can't be strings then let me know. I will also change the typing on the data argument

Reply:
The reason is that I believe wandb does not support arbitrary objects. They must be scalars or wandb supported objects

Reply:
Sounds good to me.